go-controller/pkg/metrics/metrics.go

```diff
@@ -156,6 +158,30 @@ func getCoverageShowOutputMap(component string) (map[string]string, error) {
 	return coverageShowMetricsMap, nil
 }
 
+// ovnKubeLogFileSizeMetricsUpdater updates the metrics that obtains the
+// size of ovnkube process' logfile
+func ovnKubeLogFileSizeMetricsUpdater(ovnKubeLogFileMetric *prometheus.GaugeVec,
+	stopChan <-chan struct{}) {
+	ticker := time.NewTicker(metricsUpdateInterval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			logfile := config.Logging.File
+			fileName := path.Base(logfile)
```

Comment:
nit: do we allow log file change during runtime? if not, id put this outside the loop so we dont have to do it for each iteration.

Reply:
It doesn't change. Going to move it outside.